sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Bug fix for async paged requests to use api keys
```

Comment:
I'd recommend a more "customer facing" description. Something like: "Fixed issue where async list calls could fail with a 403 error when there was too much data to retrieve."